### ISSUE TITLE
Upload Michigan/Caplan to staging/preview and QC

### ISSUE THREAD

**Issue by mbakeryo** [mlibrary/heliotrope#1554]
For more details on Caplan:
https://docs.google.com/spreadsheets/d/1dDGwW9uWouw92JTNbKkFvkCD9guPwvLNxisGewHu5l8/edit?usp=sharing
- [x] Review metadata template
- [x] Review assets
- [ ] transcripts for video and audio files
- [x] Write up summary of assets - available here: https://docs.google.com/document/d/1L9Ek0x4KfKimRUHoIVn-YDsBUzywH9IVjIIWPOamEMk/edit
- [x] Upload assets and metadata to preview
- [x] Send to Acq for ok - sent on 4/5/18
- [ ] Fix thumbnail and poster for video

**Comment by mbakeryo** [comment]
@conorom @sethaj There are a number of assets that don't have a height defined and therefore do not display on the asset page. How can the height be set? I'm assuming I can't do this through the UI. 

The figures with this problem are:
Fig05 - _Got, mentsh, un tayvl_ (_God, Man, and Devil_), Vilna, 1916
Fig06 - _Got fun nekome_ (_God of Vengeance_), Vilna, 1916
Fig18 - _Der dibek_ (_The Dybbuk_), Berlin, 1922
Fig25 -Alexander Asro and Sonia Alomis arriving in New York, 1924
Fig28 - Alexander Asro and Sonia Alomis with their son, Joseph Asro

Files can be found in this Caplan FINAL folder: https://drive.google.com/drive/folders/117tDFXiXyMGNlet3KF5w5Y3CtBISMiq4?usp=sharing

**Comment by conorom** [comment]
No UI, console only right now.

We should come up with a long term solution to this, _probably_ including exposing width/height values on image FileSets to the UI.

I bet a lot of them would be solved by configuring only one simple/robust FITS tool for images, so that might be worth doing (as well?). It doesn't take much for an inter-tool "conflict" to happen and mess things up. I know some of these in the past were caused by reversed width/height, as the tool didn't account for EXIF orientation, or did but it caused a conflict. I can think of one on staging/preview, which actually got through with the width/height reversed, and still displays in leaflet, though cropped in the supposedly-shorter dimension.

Maybe just allowing these values to be altered through the UI is the way to go, combined with the work in #1701 that lets you know there's been a failure to set the dimensions, with the caveat that this would not catch the reversed-dimension image I just mentioned.

In the past we've also exported the images to a version that got through FITS and the rest unscathed.

What makes the most sense for future workflows? All three:
- mess with FITS to reduce the occurrence
- allow setting image dimensions in the UI
- somehow flag images that didn't have their dimensions set at all

Parking lot?
